tool/docs/nodejs/NodejsDocsParser.kt

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.vaticle.typedb.driver.tool.doc.nodejs
+
+import com.vaticle.typedb.driver.tool.doc.common.*
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Element
+import picocli.CommandLine
+import picocli.CommandLine.Parameters
+import java.io.File
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.util.concurrent.Callable
+import kotlin.system.exitProcess
+
+fun main(args: Array<String>): Unit = exitProcess(CommandLine(NodejsDocParser()).execute(*args))
+
+@CommandLine.Command(name = "RustDocsParser", mixinStandardHelpOptions = true)
+class NodejsDocParser : Callable<Unit> {
+    @Parameters(paramLabel = "<input>", description = ["Input directory (for rust specify async first)"])
```

Comment:
Why is Rust mentioned here?

Reply:
do you have to redeclare the paramters in every parser? Maybe we should have a common entry point in a separate class that then calls these classes?